app/src/main/res/layout/logins_item.xml

```diff
@@ -36,6 +36,7 @@
         app:layout_constraintStart_toEndOf="@+id/favicon_image"
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintVertical_chainStyle="packed"
+        android:contentDescription="@string/text_field_for_displaying_web_address"
```

Comment:
Consider running the format on Android Studio, I think that should help reorder the XML attributes and format the attributes nicely. Otherwise, we would like to group the `android` attributes with the other `android` attributes.

Reply:
I believe the auto-format commands are CTRL+ALT+L on Windows or command+option+L on Mac

Reply:
done